format_test.go

```diff
@@ -117,6 +117,12 @@ func Test_format(t *testing.T) {
 			expectedOut:  ``,
 			errorMatcher: IsJSONObjectParse,
 		},
+		{
```

Comment:
In this test `verbosity` has `int` value. Before the change it would fail and print the same JSON.